### PR TITLE
feat(mcdu): scratchpad brightness indicator

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

Adds the brightness indicator bar that appears over the scratchpad when changing brightness.

## Screenshots (if necessary)

Same as this video but no RDY light. IRL power-off logic is not implemented at the moment as it's not really suitable for the sim input methods. We will have to come up with a solution later.

https://user-images.githubusercontent.com/9995998/147932773-cac7cf4e-7ea5-4086-8f03-7aa6a51ee924.mp4

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Check that the brightness increments nicely through each increment of the brightness keys, and the indicator appears/dissapears properly over the scratchpad.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by derl30n** [comment]
I like what I see, however I doubt that changing the brightness and "displaying" such will/should have any effect on the scratchpads data and displayed content itself.
From my current understanding the brightness indicator will "override" what ever content is displayed, however I'd expect that exact content to become visible again once the brightness indicator is vanished, hence the scratchpad content remains untouched.
Do you have actual reference on what happens if content is displayed (when changing brightness) and does the behavior change depending on the type of content displayed? (e.g. user content, type I message, type II message)

**Comment by tracernz** [comment]
The content is not overwritten, just temporarily replaced with the brightness boxes (and + sign on the right side if less than full brightness).

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js [hunk 1]
```diff
@@ -15,6 +15,12 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this._lines = [];
         this._keypad = new Keypad(this);
         this.scratchpad = null;
+        this._inout = null;
```
Comment (2hwk) [inline flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:18]:
```suggestion
        this._inOut = null;
```

Is this supposed to be ``this._inOut`` ?

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js (hunk 1 above)
Comment (tracernz) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:18]: Yes it is. Great catch!

**Pull request closed without merging**